README.md

```diff
@@ -0,0 +1,56 @@
+# Azure SDK for Python
```

Comment:
The current readme is rst, I guess this PR should remove the readme.rst as well?

Reply:
We will. I was going to check if it was in use/directly referenced by any our existing distribution packages first. I assume not.